pkg/dataencryption/snapstore.go

```diff
@@ -0,0 +1,108 @@
+package dataencryption
+
+import (
+	"crypto/sha256"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	"github.com/minio/sio"
+	"golang.org/x/crypto/hkdf"
+
+	"github.com/gardener/etcd-backup-restore/pkg/types"
+)
+
+type decoratedSnapStore struct {
+	snapstore            types.SnapStore
+	encryptionConfigPath string
+}
+
+func (r *decoratedSnapStore) readEncryptionMasterKey() string {
+	if r.encryptionConfigPath == "" {
+		return ""
+	}
+	content, err := ioutil.ReadFile(r.encryptionConfigPath)
```

Comment:
```suggestion
	content, err := os.ReadFile(r.encryptionConfigPath)
```
see this: https://go.dev/doc/go1.16#ioutil

Reply:
Thanks